### PR TITLE
doc: be consistent with the use of "sudo" - v6

### PR DESCRIPTION
**Pull request by 0xEniola**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/5720

Previous PR: https://github.com/OISF/suricata/pull/9591

Describe changes:
- Incorporate feedback from last PR.

### CODE REVIEW

File: doc/userguide/install.rst [hunk 1]
```diff
@@ -364,9 +380,5 @@ Suricata packages.
 Advanced Installation
 ---------------------
 
-If you are using Ubuntu, you can follow
-:doc:`devguide/codebase/installation-from-git`.
-
-For other various installation guides for installing from GIT and for other operating
-systems, please check (bear in mind that those may be somewhat outdated):
+Various installation guides for installing from GIT and for other operating systems are maintained at:
```
Comment (jufajardini) [inline doc/userguide/install.rst:383]: Ooooppps, we almost let a regression happen here. This was probably removed after a merge conflict.

Please, make sure we don't miss these recent changes added by https://github.com/OISF/suricata/commit/4ab4f711de1ad3424499f21a145f7f68d48c7173

**Review by jufajardini — CHANGES_REQUESTED** [review]
I'm sorry for not noticing what I just pointed out in previous reviews, sometimes, things can be quite sneaky.

I've caught up a regression of a recent change, which we must bring back ;)

(Only because changes are needed, could you please remove the period from the commit message? This isn't something we usually have. Thanks for your resilience in incorporating feedback! :) )

**Comment by 0xEniola** [comment]
> I'm sorry for not noticing what I just pointed out in previous reviews, sometimes, things can be quite sneaky.
> 
> I've caught up a regression of a recent change, which we must bring back ;)
> 
> (Only because changes are needed, could you please remove the period from the commit message? This isn't something we usually have. Thanks for your resilience in incorporating feedback! :) )

So I'm to remove the period and sync the `master` branch with OIST's right?

**Comment by jufajardini** [comment]
> > I'm sorry for not noticing what I just pointed out in previous reviews, sometimes, things can be quite sneaky.
> > I've caught up a regression of a recent change, which we must bring back ;)
> > (Only because changes are needed, could you please remove the period from the commit message? This isn't something we usually have. Thanks for your resilience in incorporating feedback! :) )
> 
> So I'm to remove the period and sync the `master` branch with OIST's right?

Remove the period from the commit message, yes, and
rebase your changes with suricata master, and make sure you keep master's changes, for that last section that I've pointed out, yep

**Pull request closed without merging**